PLNPluginSchemaMigration.inc.php

```diff
@@ -22,40 +21,41 @@ class PLNPluginSchemaMigration extends Migration {
 	 * @return void
 	 */
 	public function up() {
-		// PLN Deposit Objects
-		Capsule::schema()->create('pln_deposit_objects', function (Blueprint $table) {
-			$table->bigInteger('deposit_object_id')->autoIncrement();
-			$table->bigInteger('journal_id');
-			$table->bigInteger('object_id');
-			$table->string('object_type', 36);
-			$table->bigInteger('deposit_id')->nullable();
-			$table->datetime('date_created');
-			$table->datetime('date_modified')->nullable();
-		});
+		// Before the version 2.0.4.3, it's needed to check for a missing "export_deposit_error" field
```

Comment:
I've added an empty `upgrade.xml` to this PR, just to kickoff this migration during an upgrade through the UI.
There's some non-expected behavior, I'll write about it somewhere else :)

Reply:
I'm OK to move ahead with the empty `upgrade.xml` work-around as is; there are some good proposals on [pkp/pkp-lib#7987](https://github.com/pkp/pkp-lib/discussions/7987) but I don't think we need to solve this for 3.3.x or 3.4.0. Please go ahead!

Reply:
Ok!